src/tools/lifter/selftest_riscv.sml

```diff
@@ -156,6 +156,9 @@ val _ = print_msg "\n";
    * memory address in x2 with offset 8 *)
   val _ = riscv_test_hex_print_asm "SW x14, 8(x2)" "00E12423";
 
+  (* Usage of the zero register: ("SW x0, 0(t2)") *)
+  val _ = riscv_test_hex_print_asm "SW x0, 0(t2)" "0003A023";
```

Comment:
```suggestion
  val _ = riscv_test_hex_print_asm "SW x0, 0(x2)" "0003A023";
```
Surely the text should follow conventions with `xN` for general purpose registers?

Reply:
The rest of the file uses the bare names of the registers so I will change this, however `t2` is the [ABI mnemonic](https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/master/riscv-cc.adoc#register-convention) of `x7` (also this can be seen in the selftest log), so your suggestion is wrong. The ASM must have been copy-pasted from somewhere.